classes/class4/homework/routes/ingredients.js

```diff
@@ -0,0 +1,55 @@
+var routes = {};
+
+var mongoose = require('mongoose');
+var Ingredient = require('../models/ingredientModel.js')
+var Order = require('../models/orderModel.js')
+
+routes.home = function(req, res){
+	Ingredient.find({}).exec(function(err, allingrd) {
+		res.render('ingredients',{ingr:allingrd});
+	});
+}
+
+routes.addIngr = function(req, res){
+	var data = req.body;
+	ing = new Ingredient({
+		name: data.ingredient,
+		price: data.price,
+		stocked: true
+	});
+	ing.save(function (err) {
+		if (err) {
+			console.log('problem saving ingredient', err);
+		}
+	});
+	res.end();
+}
+
+routes.outStock = function(req, res){
+	var data=req.body;
+	var stockUp = false;
+	if (data.stocked == true) {
```

Comment:
This block is a whole lot of backwards semantics.
You can also accomplish the same thing with 
```
var stockUp = !data.stocked;
```